src/components/schema-editor/schema-editor.component.tsx

```diff
@@ -40,8 +40,12 @@ const SchemaEditor: React.FC<SchemaEditorProps> = ({
 
   const inputDummySchema = useCallback(() => {
     const dummySchema: OHRIFormSchema = {
-      encounterType: "",
+      encounterType: "Test",
```

Comment:
Something minor that crossed my mind: Assuming we manage to implement a schema validator in the future and we require this attribute, we may be force to do something like:

```js

if (isNotEmpty(rootFormNode.encounterType) && rootFormNode.encounterType !== 'Test') {
   // pass
}

```

We need to be more careful with some of these defaults. The same case applies for the `formUuid` attribute.